builtin/logical/pki/acme_wrappers.go

```diff
@@ -46,13 +47,13 @@ func (b *backend) acmeWrapper(op acmeOperation) framework.OperationFunc {
 			return nil, fmt.Errorf("ACME is disabled in configuration: %w", ErrServerInternal)
 		}
 
-		baseUrl, err := getAcmeBaseUrl(sc, r.Path)
+		acmeBaseUrl, err := getAcmeBaseUrl(sc, r.Path)
```

Comment:
Does this do anything?

Reply:
no, oops, something I missed rolling back, I had made changes to this area but rolled them back. Will remove this.

Reply:
Nah, its fine to leave too, was just curious if this actually changed anything :D